models/engine/file_storage.py

```diff
@@ -55,11 +72,11 @@ def reload(self):
                 jo = json.load(f)
             for key in jo:
                 self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
-        except:
+        except Exception:
             pass
 
     def delete(self, obj=None):
-        """delete obj from __objects if it’s inside"""
+        """delete obj from __objects if it's inside"""
```

Comment:
Updated the get and count function, added the exception command, and edited the function description for the delete function.